download_and_create_reference_datasets/v02/hail_scripts/write_dbnsfp_ht.py

```diff
@@ -176,7 +133,7 @@ def dbnsfp_to_ht(source_path, output_path, reference_genome='37', dbnsfp_version
         version=dbnsfp_version,
     )
 
-    ht.write(output_path)
+    ht.repartition(10000).write(output_path, overwrite=True)
```

Comment:
what does this do?

Reply:
Left this in by mistake! I thought hail was doing something funky so added it in for a test. I'll remove and